pkg/lightning/errormanager/errormanager.go

```diff
@@ -89,10 +89,10 @@ const (
 		CREATE TABLE IF NOT EXISTS %s.` + ConflictErrorTableName + ` (
 			task_id     bigint NOT NULL,
 			create_time datetime(6) NOT NULL DEFAULT now(6),
-			table_name  varchar(261) NOT NULL,
-			index_name  varchar(128) NOT NULL,
```

Comment:
I also found when it's record-type conflict, `'PRIMARY'` will be used to insert. So this column can also be NOT NULL